industrial_ci/src/tests/source_tests.sh

```diff
@@ -97,6 +97,23 @@ function run_clang_tidy_check {
     fi
 }
 
+function run_pylint_check {
+    local target_ws=$1
+    local pylint_args=()
+
+    ici_parse_env_array pylint_args PYLINT_ARGS
+
+    local pylint_find_pattern=()
+    # shellcheck disable=SC2016,SC2034
+    for p in $PYLINT_EXCLUDE; do pylint_find_pattern+=(-not -path "*$p*"); done
```

Comment:
What your intention here?
At first you glob it for `p` and then you add wildcards in the path as well?

How should `PYLINT_EXCLUDE` work?

Reply:
that is due to the fact that `pylint` seems to have problems with ignoring sub-folders/files - there are multiple issues on this topic
so I introduce `PYLINT_EXCLUDE: pkgA` to not include any files of that `pkgA`, i.e. `find -not -path "*pkgA*" -type f --iname "*.py"`

Reply:
I see.
So for now, we can use a non-globbing parameter together with this wildcard search.
And later, it might be better to filter out modules properly.